src/components/sync-connected-sites.tsx

```diff
@@ -241,7 +243,10 @@ const SyncConnectedSitesList = ( { selectedSite }: SyncConnectedSitesListProps )
 						<div className="flex shrink-0 justify-self-end">
 							{ isPulling && (
 								<div className="flex flex-col gap-2 min-w-44">
-									<div className="a8c-body-small">{ sitePullState.status.message }</div>
+									<div className="a8c-body-small">
+										{ importState[ connectedSite.localSiteId ]?.statusMessage ||
+											sitePullState.status.message }
```

Comment:
This might be a bit subjective, but maybe we could consider extracting out this logic for better readability (e.g. into a separate const `statusMessage`).

Reply:
That makes total sense to me, updated as part of 6afe9c99